node/src/chain_spec.rs

```diff
@@ -114,9 +114,6 @@ fn eden_testnet_genesis(
 	RuntimeGenesisConfig {
 		// Core
 		system: SystemConfig {
-			code: WASM_BINARY
-				.expect("WASM binary was not build, please build it!")
-				.to_vec(),
```

Comment:
Does it mean the they have remove the genesis code segment from chain-specs? Where has it gone?

Reply:
Code is added in `from_genesis` call